packages/base/src/formbuilder/objectform/baseform.tsx

```diff
@@ -294,3 +298,15 @@ export class BaseForm extends React.Component<IBaseFormProps, IBaseFormStates> {
 
   protected currentFormData: IDict<any> | undefined;
 }
+
+// const MyCustomWidget = (props: WidgetProps) => {
+//   return (
+//     <input
+//       type="text"
+//       className="custom"
+//       value="fucking doododoododoass"
+//       required={props.required}
+//       onChange={event => props.onChange(event.target.value)}
+//     />
+//   );
+// };
```

Comment:
I guess this is a custom color widget you wanted to introduce? Let's doododoododoass it from the file for now 😆 

```suggestion
```